demos/src/Provisioning/aws_iot_demo_provisioning_with_csr.c

```diff
@@ -364,7 +364,7 @@ static int _establishMqttConnection( const char * pIdentifier,
 
 /**
  * @brief The function that runs the Provisioning demo, for the AWS IoT generated
- * certificate from device generated CSR use-case, called by the demo runner.
+ * cert from device generated CSR use-case, called by the demo runner.
```

Comment:
I'm ambivalent about using the abbreviation in the member variable name. But I think that the comment should spell out the word for maximum clarity.

Reply:
Accidental change.

Reply:
Reverted